nucliadb_node/src/grpc/grpc_writer.rs

```diff
@@ -169,13 +170,22 @@ impl NodeWriter for NodeWriterGRPCDriver {
         &self,
         _request: Request<EmptyQuery>,
     ) -> Result<Response<ShardIds>, Status> {
-        let shard_ids = self
-            .shards
-            .list_loaded_ids()
-            .await
-            .into_iter()
-            .map(|shard_id| ShardId { id: shard_id })
-            .collect();
+        let mut entries = fs::read_dir(self.shards.shards_path.clone()).await?;
+        let mut shard_ids = Vec::new();
+        while let Some(entry) = entries.next_entry().await? {
+            let entry_path = entry.path();
+            if entry_path.is_dir() {
+                shard_ids.push(ShardId {
+                    id: entry_path
+                        .file_name()
+                        .unwrap()
+                        .to_str()
+                        .unwrap()
+                        .to_string(),
+                })
```

Comment:
To avoid unwrapping on strings taken from disk I would do the following:
```rust
if entry_path.is_dir() {
         let Some(id) = entry_path.file_name().map(|s| s.to_str().map(String::from)) else {
                continue;
          };
          shard_ids.push(ShardId { id });
}
```